spec/lib/schema_consistency_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'spec_helper'
+require 'govuk_schemas'
+require "finder_schema_converter"
+
+RSpec.describe "the finder schemas" do
+  def specialist_document_schema_definitions
+    GovukSchemas::Schema.find(publisher_schema: "specialist_document")["definitions"]
+  end
+
+  context "all facets on finder schema should appear as metadata in the specialist document schema" do
+    FinderSchemaConverter.except_ignored_finders.each do |finder_schema|
```

Comment:
Yeah that sounds reasonable, since the test is the only thing making use of it, and it would make it a bit more obvious what's being excluded.